yarn-project/noir-libs/noir-aztec/src/abi.nr

```diff
@@ -246,15 +260,42 @@ struct PublicCircuitPublicInputs {
     new_commitments: [Field; MAX_NEW_COMMITMENTS_PER_CALL],
     new_nullifiers: [Field; crate::abi::MAX_NEW_NULLIFIERS_PER_CALL],
     new_l2_to_l1_msgs: [Field; crate::abi::MAX_NEW_L2_TO_L1_MSGS_PER_CALL],
-    commitment_trees_roots: CommitmentTreesRoots,
+    commitment_trees_roots: CommitmentTreesRoots, // TODO: This is not present in cpp or ts, do we need to include it?
```

Comment:
@Maddiaa0 I noted this field was added in Noir but has no counterpart in cpp. Should we add it there as well, so the two structs don't diverge?